### DISCUSSION TITLE
Restructuring the Keptn Enhancement Proposals Repo & Workflow

### DISCUSSION THREAD

**Discussion by oleg-nenashev** [keptn/keptn#9082] [Ideas]
Hi all. I would like to change how our KEPs are stored. Right now we have a bunch of open KEP pull requests that include incomplete or never-started ideas. Let's admit most of them will never be completed, but at the same time they still include the valuable knowledge are we would like to preserve in the project. The current approach also confuses users that see a lot of open never-ever KEPs with unclear status. Especially things like GitOps Operator and Authorization support that are highly anticipated but nobody actively works on.

I would like to change the approach to KEP management, and basically adopt what we did in Jenkins instead of the OpenTelemetry pattern. See [this repo](https://github.com/jenkinsci/jep) for the listing.

NOTE: The KEP decision making process is outside the scope of this discussion for now, I focus only on the structure

## Suggested changes

- Merge all KEP pull requests that contain valuable information. Close the drafts that do not have enough info & not started
- Create GitHub issues for those KEPs that we want to keep on the roadmap
- Add status tables in the beginning of each KEP that would make the status and plans explicit. Also I would like to put "help-wanted" 
- Call for participation for those items that still make sense for end users of upcoming Keptn 1.x
- Add at least think of KEPs to the README page, similarly to how Jenkins does it
- Update the contributing guidelines accordingly

## References

- [Keptn Enhancement Proposals](https://github.com/keptn/enhancement-proposals)
- [Jenkins Enhancement Proposals](https://github.com/jenkinsci/jep)

**Comment by agardnerIT** [comment]
How do future KEPs get raised? As Issues? If so I'm a fan.

**Reply by oleg-nenashev** [reply]
Yes. We create an issue as a placeholder to keep the KEP number and to track references/status. And the PRs are just used top update the content.

Bonus points: With that RFEs and GitHub Discussions can be easily converted to KEPs
